src/attr/_funcs.py

```diff
@@ -71,12 +71,7 @@ def has(cl):
 
     :rtype: :class:`bool`
     """
-    try:
-        fields(cl)
-    except ValueError:
-        return False
-    else:
-        return True
+    return hasattr(cl, "__attrs_attrs__")
```

Comment:
`return getattr(cl, "__attrs_attrs__", None) is not None` [please](https://hynek.me/articles/hasattr/). :)

We also may start renaming all those `cl` s in in `cls` s.  Dunno what I’ve been thinking. oO